### DISCUSSION TITLE
why i cannot use in description symbols like "*" and "#"?

### DISCUSSION THREAD

**Discussion by onlineapps-cloud** [btcpayserver/btcpayserver#5467] [Technical Support] [answered]
hello, why i cannot use in description symbols like "*" and "#"?
i need to put link that contain "#" but i unable to do this!
![image](https://github.com/btcpayserver/btcpayserver/assets/69077240/8cf78a76-95ac-4b32-a531-481002f9352b)

**Comment by dennisreimann** [comment, marked as answer]
It's a legacy requirement, by now you can use them and with v1.12 we'll remove that check. Until then, you can edit the description via the raw editor I think.

**Comment by onlineapps-cloud** [comment]
i tried to do in  © BTCPay Server v1.11.7+a921504bc and not working.
in raw editor in json i don't have field description.
upd, description field exist, but it's empty, in normal editor i have some description, why in raw editor description not appear?
upd2, my mistake, i will try with raw editor, thanks.
Upd3, all work as expected using raw editor, thanks.
